charts/incubator/twofauth/values.yaml

```diff
@@ -0,0 +1,148 @@
+image:
+  repository: tccr.io/truecharts/2fauth
+  pullPolicy: IfNotPresent
+  tag: v4.2.2@sha256:0383cb30609aa4cb60fd00b7a15c7f8e18533c18714144d498fcc2ea0744297f
+
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        protocol: http
+        targetPort: 8000
+        port: 8000
+
+twofauth:
+  app:
+    name: "2FAuth"
+    debug: false
+    site_owner: "mail@example.com"
+    url: "http://localhost"
+    # daily | single
+    log_channel: "daily"
+    # debug | info | notice | warning | error | critical | alert | emergency
+    log_level: "notice"
+    trusted_proxies:
+      - "*"
+  mail:
+    driver: smtp
+    host: smtp.mailtrap.io
+    port: 587
+    from: changeme@example.com
+    user: changeme@example.com
+    pass: changeme
+    # TLS | STARTTLS | SSL
+    encryption: STARTTLS
+    from_name: 2FAuth
+    from_address: changeme@example.com
+  auth:
+    # web-guard | reverse-proxy-guard
+    guard: web-guard
+    # show if guard uses reverse-proxy-guard
+    proxy_header_for_user: ""
+    proxy_header_for_email: ""
+    proxy_logout_url: ""
+  webauthn:
+    name: 2FAuth
+    id: ""
+    icon: ""
+    # required | preferred | discouraged
+    user_verified: preferred
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            APP_KEY:
+              secretKeyRef:
+                name: twofauth-secrets
+                key: APP_KEY
+            APP_ENV: local
+            # If you're looking for performance improvements, you could install memcached.
+            CACHE_DRIVER: file
+            SESSION_DRIVER: file
```

Comment:
This is probably a Laravel app?
Those could probably be set to `redis`

Reply:
that was the default but ill change it to redis.